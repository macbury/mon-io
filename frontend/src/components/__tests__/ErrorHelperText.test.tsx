import React from 'react'
import { shallow } from 'enzyme'
import snapshot from 'enzyme-to-json'

import ErrorHelperText from '../ErrorHeplerText'

it('renders graphQLError and errors correctly', () => {
  const props = {
    error: new Error('Graphql nasty error'),
  }

  const component = shallow(<ErrorHelperText {...props}/>)
  expect(snapshot(component)).toMatchSnapshot()
})

it('renders nothing correctly', () => {
  const props = { }

  const component = shallow(<ErrorHelperText {...props}/>)
  expect(snapshot(component)).toMatchSnapshot()
})