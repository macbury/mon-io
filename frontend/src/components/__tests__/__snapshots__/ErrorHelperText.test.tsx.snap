// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders graphQLError and errors correctly 1`] = `
<withTheme(HelperText)
  type="error"
>
  <Component />
</withTheme(HelperText)>
`;

exports[`renders nothing correctly 1`] = `""`;
